observations/marine/sst_generic.yaml.j2

```diff
@@ -44,3 +45,20 @@
     where:
     - variable: {name: GeoVaLs/distance_from_coast}
       minvalue: 50e3
+
+  obs post filters:
+  - filter: Background Check
+    absolute threshold: 5.0
+  - filter: Perform Action  # Palmico Sound
```

Comment:
This would override the results of background check; if you'd like to keep background check QC (even at Pamlico Sound), you can move `Perform Action` filter above `Background check`.

Reply:
Dang ... You told me the other day but I forgot to update the yaml. Thanks @shlyaeva .